bin/nameserver

```diff
@@ -45,6 +45,8 @@ if __name__ == '__main__':
                         action="store_true")
     parser.add_argument("--no-multicast", help="disable multicasting",
                         action="store_true")
+    parser.add_argument("-r", "--restrict-to-localhost", help="restrict incomming processes to localhost",
```

Comment:
Should the help text have "connections" instead of "processes"? Or a more clear "accept connections only from localhost"?

How about using a bit shorter name for the option `-L`, `--local-only`?

Reply:
Sounds good, I'm on it